pkgs/development/python-modules/aioinflux/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, pythonAtLeast
+, fetchFromGitHub
+, pytz
+, aiohttp
+, ciso8601
+, pytest
+, pytest-asyncio
+, pytestcov
+, pyyaml
+, flake8
+, pep8-naming
+, pygments
+, pandasSupport ? true
+, pandas
+, numpy
+# , cacheSupport ? false   # TODO: Optional Cache Support
+# , aioredis
+# , lz4
+}:
+
+buildPythonPackage rec {
+  pname = "aioinflux";
+  version = "0.9.0";
+
+  disabled = !(pythonAtLeast "3.6");
```

already a function for this :)
```suggestion
  disabled = pythonOlder "3.6");
```

---

Looks like I chose the wrong package to copy from, `pythonOlder` is much clearer. 

Is this an actual commit suggestion, or are you just being helpful? Accepting your commit suggestion will spoil my perfect commit messages.

There's a couple other packages that didn't get the memo:
```sh
$ rg 'disabled = !\(?pythonAtLeast'  -l
aioprocessing/default.nix
jupyter-repo2docker/default.nix
moviepy/default.nix
nest-asyncio/default.nix
aioinflux/default.nix
```
Maybe we could have a separate commit that cleans them up?

---

no, for nixpkgs, the commits should make sense when you cherry-pick them to another branch. We should consolidate as many package changes into their related commits.

But, if you had one commit that fixed a build, and another that bumped it... then it might make sense to do 2 commits, but in this case as the first addition, it should all be rolled up into one.

to append changes to the last commit:
```
# do changes ...
git add pkgs/
git commit --amend --no-edit
git push <remote> <branch> --force
```